src/assets/styles/20-tools/_mixins/_grid.scss

```diff
@@ -189,6 +189,11 @@
 
   $column-gap: false;
   $row-gap: false;
+  $column: 1;
+  $row: 1;
+  $counter: 1;
+
+  grid-auto-flow: $autoflow;
```

Comment:
FYI this is fine and doesn't break anything (grid-autoflow isn't currently used by anything, but I don't think it _will_ break if/when it is used)